components/match2/wikis/wildrift/match_group_input_custom.lua

```diff
@@ -438,28 +438,39 @@ function matchFunctions.getPlayersOfTeam(match, oppIndex, teamName, playersData)
 	-- match._storePlayers will break after the first empty player. let's make sure we don't leave any gaps.
 	playersData = Json.parseIfString(playersData) or {}
 	local players = {}
-	for playerIndex = 1, _MAX_NUM_PLAYERS do
-		-- parse player
-		local player = Json.parseIfString(match['opponent' .. oppIndex .. '_p' .. playerIndex]) or {}
-		player.name = player.name or playersData['p' .. playerIndex]
-			or Variables.varDefault(teamName .. '_p' .. playerIndex)
-		player.flag = player.flag or playersData['p' .. playerIndex .. 'flag']
-			or Variables.varDefault(teamName .. '_p' .. playerIndex .. 'flag')
-		player.displayname = player.displayname or playersData['p' .. playerIndex .. 'dn']
-			or Variables.varDefault(teamName .. '_p' .. playerIndex .. 'dn')
-
-		if String.isNotEmpty(player.name) then
-			player.name = mw.ext.TeamLiquidIntegration.resolve_redirect(player.name):gsub(' ', '_')
-		end
 
-		if not Table.isEmpty(player) then
-			table.insert(players, player)
-		end
+	-- need unlimited import here so we do not cut off players due to TeamCards storing players weirdly
+	local playerIndex = 1
+	local player = matchFunctions.parsePlayer(match, oppIndex, teamName, playersData, playerIndex)
+
+	while not Table.isEmpty(player) do
+		table.insert(players, player)
+		playerIndex = playerIndex + 1
+		player = matchFunctions.parsePlayer(match, oppIndex, teamName, playersData, playerIndex)
 	end
```

Comment:
```suggestion
	local players = Array.mapIndexes(function(playerIndex)
		return matchFunctions.parsePlayer(match, oppIndex, teamName, playersData, playerIndex)
	end)
```

Reply:
And update `parsePlayer` to return nil instead of empty table when there is none